crates/containerd-shim-wasm/src/container/context.rs

```diff
@@ -10,10 +10,6 @@ pub trait RuntimeContext {
     // path to the entrypoint executable.
     fn args(&self) -> &[String];
 
-    // ctx.entrypoint() returns the entrypoint path from arguments on the runtime
-    // spec process field.
-    fn entrypoint(&self) -> Option<&Path>;
-
     // ctx.wasi_entrypoint() returns a `WasiEntrypoint` with the path to the module to use
```

Comment:
**nit:** This comment still needs to be updated to reflect the new signature and document the new behaviour.